proto/osmosis/concentrated-liquidity/pool-model/tx.proto

```diff
@@ -17,6 +17,11 @@ message MsgCreateConcentratedPool {
   string denom0 = 2 [ (gogoproto.moretags) = "yaml:\"denom0\"" ];
   string denom1 = 3 [ (gogoproto.moretags) = "yaml:\"denom1\"" ];
   uint64 tick_spacing = 4 [ (gogoproto.moretags) = "yaml:\"tick_spacing\"" ];
+  string precision_factor_at_price_one = 5 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.moretags) = "yaml:\"precision_factor_at_price_one\"",
+    (gogoproto.nullable) = false
+  ];
```

Comment:
This should be a native int. The realistic range is small, our int types can't support any value above 128 here lol

Reply:
TIL you cant use in8 or int16 in protobufs. I ended up utilizing int32

Reply:
When I made it an int32, I end up just having to transform it to and sdk.Int in the functions in order to multiply / divide it / power with other sdk.Ints, so it feels like this is more trouble than it is worth. I am reverting for now, lmk if you disagree